pydantic_ai_slim/pydantic_ai/_thinking_part.py

```diff
@@ -17,20 +14,20 @@ def split_content_into_text_and_thinking(content: str) -> list[ThinkingPart | Te
     """
     parts: list[ThinkingPart | TextPart] = []
 
-    start_index = content.find(START_THINK_TAG)
+    start_index = content.find(thinking_tags[0])
```

Comment:
Instead of repeating `[0]` and `[1]`, let's add `start_tag, end_tag = thinking_tags` to as variables here